server/data/licenceApiClient.test.ts

```diff
@@ -239,30 +239,25 @@ describe('Licence API client tests', () => {
     it('Should pass parameters to sort the matched licences', async () => {
       post.mockResolvedValue([{ licenceId: 1, prisonCode: 'MDI' } as LicenceSummary])
 
-      const result = await licenceApiClient.matchLicences(
-        [LicenceStatus.IN_PROGRESS],
-        ['MDI'],
-        [1],
-        ['ABC1234'],
-        [],
-        'conditionalReleaseDate',
-        'DESC',
-        { username: 'joebloggs' } as User,
-      )
+      const result = await licenceApiClient.matchLicences({
+        statuses: [LicenceStatus.IN_PROGRESS],
+        nomisIds: ['ABC1234'],
+        pdus: [],
+        sortBy: 'conditionalReleaseDate',
+        user: { username: 'joebloggs' } as User,
+      })
```

Comment:
moving over to object args and removing staffIds, prisonCodes and sortOrder as not used